imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -136,100 +134,38 @@ class CaseExplorer extends Component {
   )
   render () {
     const { isLoading, dispatch, match, caseList, allNotifications, unreadNotifications } = this.props
-    const { showOpen, expandedUnits, assignedToMe } = this.state
+    const { filterStatus, expandedUnits, myInvolvement } = this.state
     if (isLoading) return <Preloader />
-    const caseGrouping = this.makeCaseGrouping(caseList, showOpen, assignedToMe, allNotifications, unreadNotifications)
+    const caseGrouping = this.makeCaseGrouping(caseList, filterStatus, myInvolvement, allNotifications, unreadNotifications)
     return (
       <div className='flex flex-column roboto overflow-hidden flex-grow h-100 relative'>
         <div className='bb b--black-10 overflow-auto flex-grow flex flex-column bg-very-light-gray'>
-          <div className='flex pl3 pv3 bb b--very-light-gray bg-white'>
-            <div
-              onClick={() => this.handleStatusClicked(true)}
-              className={'f6 fw5 ph2 ' + (showOpen ? 'mid-gray' : 'silver')}
-            >
-              Open
-            </div>
-            <div
-              onClick={() => this.handleStatusClicked(false)}
-              className={'f6 fw5 ml4 ph2 ' + (!showOpen ? 'mid-gray' : 'silver')}
-            >
-              Closed
-            </div>
-            <div
-              onClick={() => this.handleAssignedClicked()}
-              className={'f6 fw5 ml4 ph2 ' + (assignedToMe ? 'mid-gray' : 'silver')}
-            >
-              Assigned To Me
-            </div>
-          </div>
+          <FilterRow
+            filterStatus={filterStatus}
+            myInvolvement={myInvolvement}
```

Comment:
Here the name "myInvolvement" makes sense, because it's in the context of the new generic component you created, but you should rename the property "handleAssignedClicked" to something else so it will conform with "myInvolvement"